app/src/components/Fields/TimeField.js

```diff
@@ -3,10 +3,16 @@ import PropTypes from "prop-types";
 import TextFieldMaterial from "@material-ui/core/TextField";
 
 const TimeField = (props) => {
+<<<<<<< HEAD
+    const { id, label, formik } = props;
+  return (
+  <TextFieldMaterial
+=======
     const { id, label, formik, isRequired } = props;
   return (
   <TextFieldMaterial
     required={isRequired}
+>>>>>>> 7a9edf7cd01fab28342d7d547d47ffdbd7325a7a
```

Comment:
ekk, looks like merge hell happened. Recommend doing a new branch off of develop, and cherry-pick your changes into it.